unrpyc.py

```diff
@@ -187,64 +193,116 @@ def worker(t):
+        dest="try_harder",
+        action="store_true",
+        help="Tries some workarounds against common obfuscation methods. This is a lot slower.")
+
+    ap.add_argument(
+        '-d',
+        '--dump',
+        dest='dump',
+        action='store_true',
+        help="Instead of decompiling, pretty print the ast to a file")
+
+    ap.add_argument(
+        '-p',
+        '--processes',
+        dest='processes',
+        action='store',
+        type=int,
+        choices=list(range(1, cc_num)),
+        default=cc_num - 1 if cc_num > 2 else 1,
+        help="Use the specified number or processes to decompile."
+        "Defaults to the amount of hw threads available minus one, disabled when muliprocessing "
+        "unavailable is.")
+
+    ap.add_argument(
+        '-t',
+        '--translation-file',
+        dest='translation_file',
+        action='store',
+        default=None,
+        help="Use the specified file to translate during decompilation")
+
+    ap.add_argument(
+        '-T',
+        '--write-translation-file',
+        dest='write_translation_file',
+        action='store',
+        default=None,
+        help="Store translations in the specified file instead of decompiling")
+
+    ap.add_argument(
+        '-l',
+        '--language',
+        dest='language',
+        action='store',
+        default='english',
+        help="If writing a translation file, the language of the translations to write")
+
+    ap.add_argument(
+        '--comparable',
+        dest='comparable',
+        action='store_true',
+        help="Only for dumping, remove several false differences when comparing dumps. "
+        "This suppresses attributes that are different even when the code is identical, such as "
+        "file modification times. ")
+
+    ap.add_argument(
+        '--no-pyexpr',
+        dest='no_pyexpr',
+        action='store_true',
+        help="Only for dumping, disable special handling of PyExpr objects, instead printing them "
+        "as strings. This is useful when comparing dumps from different versions of Ren'Py. It "
+        "should only be used if necessary, since it will cause loss of information such as line "
+        "numbers.")
+
+    ap.add_argument(
+        '--no-init-offset',
+        dest='init_offset',
+        action='store_false',
+        help="By default, unrpyc attempt to guess when init offset statements were used and insert "
+        "them. This is always safe to do for ren'py 8, but as it is based on a heuristic it can be "
+        "disabled. The generated code is exactly equivalent, only slightly more cluttered.")
+
+    ap.add_argument(
+        '--register-sl-displayable',
+        dest="sl_custom_names",
+        type=str,
+        nargs='+',
+        help="Accepts mapping separated by '=', "
+        "where the first argument is the name of the user-defined displayable object, "
+        "and the second argument is a string containing the name of the displayable,"
```

Comment:
This is missing a space at the end. might've been that way originally, but mind fixing it as well?

Reply:
I let the one in L302 as i thought you want propably the parens direct on the word.

Reply:
Ah, I just missed that one 😅